src/transformers/models/superpoint/image_processing_superpoint.py

```diff
@@ -107,6 +113,8 @@ class SuperPointImageProcessor(BaseImageProcessor):
         rescale_factor (`int` or `float`, *optional*, defaults to `1/255`):
             Scale factor to use if rescaling the image. Can be overriden by `rescale_factor` in the `preprocess`
             method.
+        do_grayscale (`bool`, *optional*, defaults to `True`):
```

Comment:
Also kind reminder re this change 🤗 we should make it as False by default for Supoerpoint for backward compatibility

Reply:
I'm not sure I understand this statement as SuperPoint takes grayscaled images as input, so I think this attribute should be set to True by default in the ImageProcessor no ?

Reply:
Ok! Is it set to `False` in configs on the Hub?

Reply:
No it is set to `True`. Can you explain what you mean by backward compatibility in this context ?